examples/scopes/scopes.go

```diff
@@ -4,30 +4,30 @@ import (
 	"fmt"
 )
 
-// Scope is an important concept to understand for any programming language
+// Scope is an important concept to understand on any programming language.
 // It refers to where a variable is available / accessible.
 // If an variable is not accesible in a location (within a scope) undefined will be returned
-// when we try access it.
+// when trying to access it.
 // Read the code below and see if you understand this concept, when you run it you will recieve an
-// error saying innerLoopCat is undefined. Try alter the code
-// so it will run - there are a few ways to do this so it's up to you to
-// see what works :)
+// error saying innerLoopCat is undefined. Try to alter the code
+// to make it work - there are a few ways to do this so it's up to you to
+// see what works out :)
 
 var (
 	globalCat = "Henry"
-	// A global variable such as this, is declared at the highest level and accessible
+	// A global variable, such as this, is declared at the highest level and accessible
 	// anywhere in the package.
 	ExportableGlobalCat = "Vanessa"
-	// If the variable has a capital letter at the start this means it can be exported and
-	// if we import the package elsewhere we can acess it from other packages.
+	// If the variable has a capital letter at the start this means it can be "exported", means
+	// if we import the package elsewhere we can access it from other packages.
 	// For this workshop we will not cover this but you can read more about it here:
-	//
+	// <LINK MISSING>
```

Comment:
@sleepypioneer Added this here - in case you remember which link was feel free to add:)